src/test/java/SetTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 사이즈를 확인한다")
+    void size() {
+        assertThat(numbers.size()).isNotEqualTo(4);
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @DisplayName("Set의 구성요소를 확인한다")
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
```

Comment:
ParameterizedTest의 name속성 값을 통해서도 DisplayName을 나타낼수 있습니다.
추가로 참고만 하시라고 알려드립니다~
```suggestion
@ParameterizedTest(name = "numbers 안에 숫자 {0} 값의 존재유무 테스트하는 케이스"
@ValueSource(ints = {1, 2, 3})
```
위에처럼 작성하시게 되면 {0}에 1, 2, 3이 순차적으로 맵핑되면서 DisplayName을 완성시켜줍니다.